conf/deployment/aws/gp3_csi_storage_class.yaml

```diff
@@ -0,0 +1,4 @@
+---
+# This config is for AWS deployment with gp3 CSI storage class provisioning
+DEPLOYMENT:
+  gp3_csi: True
```

Comment:
thinking maybe it would be better to call the new key customized_deployment_storage_class and set it to the name of the actual storage class, so in this case - `gp3-csi`

Reply:
Will do.